corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -124,12 +128,18 @@ public int printTable(String namespace, String tablename) {
 
         CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> table = getTable(namespace, tablename);
         int size = table.size();
-        for (CorfuDynamicKey entry : table.keySet()) {
-            builder = new StringBuilder("\nKey:\n" + entry.getKey())
-                .append("\nPayload:\n" + table.get(entry).getPayload())
-                .append("\nMetadata:\n" + table.get(entry).getMetadata())
-                .append("\n====================\n");
-            log.info(builder.toString());
+        final int batchSize = 50;
```

Comment:
please make batchSize configurable

Reply:
this is an internal setting that should not be tunable but I will expose it anyway, thanks

Reply:
actually just had a conversation with the folks using this, we feel exposing this to the browser can cause a lot more confusion than help. this is really an internal batching for RocksDB partitions.